tests/components/demo/test_notify.py

```diff
@@ -46,104 +61,48 @@ def record_calls(*args):
     return record_calls
 
 
-@pytest.fixture(name="mock_demo_notify")
-def mock_demo_notify_fixture():
-    """Mock demo notify service."""
-    with patch("homeassistant.components.demo.notify.get_service", autospec=True) as ns:
-        yield ns
-
-
-async def setup_notify(hass):
-    """Test setup."""
-    with assert_setup_component(1, notify.DOMAIN) as config:
-        assert await async_setup_component(hass, notify.DOMAIN, CONFIG)
-    assert config[notify.DOMAIN]
-    await hass.async_block_till_done()
-
-
-async def test_no_notify_service(
-    hass: HomeAssistant, mock_demo_notify, caplog: pytest.LogCaptureFixture
```

Comment:
Are these things tested in the notify integration? Otherwise we need to test it there.

Reply:
I'll remove this.

Reply:
I added the tests to the notify entity platform tests: https://github.com/home-assistant/core/pull/115457

Reply:
I was mostly thinking about the legacy service. We still need to have test coverage for that.

Reply:
> I was mostly thinking about the legacy service. We still need to have test coverage for that.

See https://github.com/home-assistant/core/pull/115494